src/x509.c

```diff
@@ -12105,7 +12158,11 @@ int wolfSSL_X509_NAME_print_ex(WOLFSSL_BIO* bio, WOLFSSL_X509_NAME* name,
                 return WOLFSSL_FAILURE;
             }
             tmpSz = len + nameStrSz + 1; /* 1 for '=' */
-            if (bio->type != WOLFSSL_BIO_FILE)
+            if (bio->type != WOLFSSL_BIO_FILE 
+        #if defined(WOLFSSL_QT)
+            && bio->type != WOLFSSL_BIO_MEMORY
```

Comment:
This change also fixes other builds printing out of an X509 name when using a BIO_MEMORY BIO. Please test and remove the WOLFSSL_QT macro guard around it.

Reply:
Checked differences to openssl print-out. there are a few differences still, but those diff seem to exist even before 5.5.0. Will check if qt unit test is ok after changes.

Reply:
Qt unit test becomes failure. Will look at the failure.